azure-pipelines.yml

```diff
@@ -0,0 +1,42 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - "*"
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+container:
+  image: sonicdev-microsoft.azurecr.io:443/sonic-slave-buster:latest
+
+resources:
+  repositories:
+    - repository: SONiCSWSSCommon
+      type: github
+      name: Azure/sonic-swss-common
+      endpoint: Pterosaur
```

Comment:
can we not use personal?

Reply:
Yes, it will be replaced if this PR can work in my private AzurePipeline. Right now, this is just a test version.